game/core/src/main/java/com/gg/game/view/Assets.java

```diff
@@ -3,49 +3,64 @@
 import com.badlogic.gdx.Gdx;
 import com.badlogic.gdx.graphics.Texture;
 
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
 /**
  * Handles textures for objects.
  */
 class Assets {
+    private static final String imgMap_Path = "images/";
```

Comment:
```suggestion
    private static final String imagesPath = "images/";
```

Or really anything else not in both camel/snake case.